src/coreclr/jit/targetloongarch64.cpp

```diff
@@ -78,14 +78,14 @@ ABIPassingInformation LoongArch64Classifier::Classify(Compiler*    comp,
         {
             assert(!structLayout->IsBlockLayout());
 
-            uint32_t             floatFlags;
-            CORINFO_CLASS_HANDLE typeHnd = structLayout->GetClassHandle();
+            FpStructInRegistersInfo fpInfo;
+            CORINFO_CLASS_HANDLE    typeHnd = structLayout->GetClassHandle();
 
-            floatFlags = comp->info.compCompHnd->getLoongArch64PassStructInRegisterFlags(typeHnd);
+            fpInfo = comp->GetPassFpStructInRegistersInfo(typeHnd);
 
-            if ((floatFlags & STRUCT_HAS_FLOAT_FIELDS_MASK) != 0)
+            if (fpInfo.flags != UseIntCallConv)
```

Comment:
```suggestion
            if (fpInfo.flags != FpStruct::UseIntCallConv)
```

Reply:
to fix the errors:
```
/home/qiao/work_qiao/runtime/src/coreclr/jit/targetloongarch64.cpp:86:33: error: use of undeclared identifier 'UseIntCallConv'; did you mean 'FpStruct::UseIntCallConv'?
            if (fpInfo.flags != UseIntCallConv)
                                ^~~~~~~~~~~~~~
                                FpStruct::UseIntCallConv
/home/qiao/work_qiao/runtime/src/coreclr/inc/corinfo.h:370:9: note: 'FpStruct::UseIntCallConv' declared here
        UseIntCallConv = 0, // struct is passed according to integer calling convention
        ^
1 error generated.
```

Reply:
Thank you! I forgot the `using namespace FpStruct' like I did in targetriscv64.cpp.

BTW, is there a publicly available docker image that can build LoongArch64 .NET? Something like `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-cross-riscv64` for RISC-V.

Reply:
Sorry, there is no OS-image on the upstream.
The image of the Debian and Alpine is still on progressing, but only few left before the upstream issue LA64's image and I think it is very near.

Thanks again for your contribution to LA64.